examples/sound/portaudio-sine.bas

```diff
@@ -0,0 +1,82 @@
+''
+'' PortAudio Sine wave example
+''
+'' Requires libportaudio2
+
+#include "portaudio.bi"
+
+Const Pi as Double = 3.14159265358979323846
+Const SampleRate as Integer = 44100
+Const SineSamples as Integer = SampleRate / 10
+
+Type PlayPayload
+    frequency As Single
+End Type
+
+Function PlayCallback (InputBuff as Const Any Ptr, OutputBuff as Any Ptr, _
```

Comment:
On windows, the calling the convention will default to `stdcall` but the `portaudio.bi` API expects `cdecl`
This callback function should be defined with `cdecl` calling convention as in `Function PlayCallback cdecl ( ...` for portability to windows.